Gemfile

```diff
@@ -108,6 +107,9 @@ group :test do
   # Generates a test coverage report on every `bundle exec rspec` call. We use
   # the output to feed CodeClimate's stats and analysis
   gem "simplecov", require: false
+  # A gem providing "time travel" and "time freezing" capabilities, making it
+  # dead simple to test time-dependent code.
+  gem "timecop"
```

Comment:
nooooooo againnnn :D :D shall we go with the https://andycroll.com/ruby/replace-timecop-with-rails-time-helpers-in-rspec/? :D :D I will be happy to then do the housekeeping on the other projects :)

Reply:
I'm gonna stick with timecop. I don't think its a major issue using it, and I'd rather do the conversion as a separate exercise rather than hold this one up.

Using timecop here makes it consistent with the other projects, and then we can plan in the conversation to time-helpers happens across projects in sync.